packages/icon/src/Icon.spec.tsx

```diff
@@ -1,43 +1,52 @@
 import React from 'react';
 import path from 'path';
 import fs from 'fs';
+import { createHash } from 'crypto';
 import { toJson } from 'xml2json';
-import { render, cleanup } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import { typeIs } from '@leafygreen-ui/lib';
 import { SVGR } from './types';
 import { createIconComponent, glyphs } from '.';
-import createGlyphComponent, { getGlyphTitle } from './createGlyphComponent';
+import { getGlyphTitle } from './glyphCommon';
+import createGlyphComponent from './createGlyphComponent';
 import EditIcon from '@leafygreen-ui/icon/dist/Edit';
 
-afterAll(cleanup);
+function getBaseName(filePath: string): string {
+  return path.basename(filePath, path.extname(filePath));
+}
 
-describe('packages/Icon/glyphs/', () => {
-  const glyphPaths = fs
-    .readdirSync(path.resolve(__dirname, './glyphs'))
-    /**
-     * './glyphs/' contains an index.ts file, so we filter out
-     * anything that's not an svg.
-     */
-    .filter(path => /.*\.svg$/.test(path));
+const glyphPaths = fs
+  .readdirSync(path.resolve(__dirname, './glyphs'))
+  /**
+   * './glyphs/' contains an index.ts file, so we filter out
+   * anything that's not an svg.
+   */
+  .filter(path => /.*\.svg$/.test(path))
+  .map(fileName => path.resolve(__dirname, './glyphs/', fileName));
+
+const generatedFilesDirectory = path.resolve(__dirname, './generated');
+const baseNameToGeneratedFilePath: Record<string, string> = {};
 
+fs.readdirSync(generatedFilesDirectory).forEach(filePath => {
```

Comment:
May be nice to move this out of top of the module, so it's not done on immediate import of the file.

Reply:
Ah...I see cuz of the tests you have. W/e, not a big deal.